packages/react-app/components/Fonts.jsx

```diff
@@ -0,0 +1,26 @@
+import { Global } from "@emotion/react";
+
+const Fonts = () => (
+  <Global
+    styles={`
+      @font-face {
+        font-family: 'PPNeueMachina';
+        font-style: normal;
+        font-weight: 600;
+        font-display: swap;
+        src: url('/fonts/PPNeueMachina-InktrapSemibold.woff2') format('woff2');
+        unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
```

Comment:
This is not necessary, as ChatGPT mentioned here : 

<details>

<summary>
ChatGPT reply : 
</summary>

> what is unicode range ?  and is it necessary  ?

The unicode-range property in CSS specifies which characters a particular font face should cover, allowing browsers to download only the necessary font resources. This is useful for performance optimization, particularly when using large fonts that support many characters.

It's not strictly necessary, but using unicode-range can improve page load times and efficiency, especially for websites that use multiple or large font files. It's particularly beneficial when you need only a subset of a font's full character set.

</details>

Just kept it, since it was also used in Chakar docs and also nice to have ? 

<details>

<summary>
Detailed answer by chatGPT : 
</summary>

The `unicode-range` property in CSS is used to define the specific range of Unicode characters that a particular font face should handle. This is particularly useful when you're using web fonts, as it allows the browser to download font files selectively based on the characters actually needed by the webpage. This can significantly optimize page loading times and performance, especially when dealing with large font files that support many characters.

Here's a breakdown of what the `unicode-range` property does:

1. **Selective Loading**: If a webpage only uses a subset of characters (for example, standard Latin characters, numbers, and some punctuation), specifying a `unicode-range` that covers just those characters means the browser won't download additional font files that cover characters not used on the page.

2. **Fallback Mechanism**: It allows the browser to fall back to a different font if the characters needed are not covered by the font being used. For example, if a user enters text in a form in Japanese, but the specified font only supports Latin characters, the browser will use a different font that does support Japanese for those characters.

3. **Efficiency**: It helps in reducing bandwidth usage and speeds up the web experience by avoiding unnecessary font downloads.

In the code you shared, the `unicode-range` includes a variety of character sets, such as:

- `U+0000-00FF`: Basic Latin, which covers the standard alphabet and common punctuation.
- `U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC`: These are specific characters outside the basic set, like ligatures, and modified letters.
- `U+2000-206F`: General Punctuation.
- `U+2074, U+20AC, U+2122`: Includes the euro symbol, trade mark sign, and other specific characters.
- `U+2191, U+2193`: Directional arrows.
- `U+2212, U+2215`: Mathematical operators.
- `U+FEFF, U+FFFD`: Special purpose characters like the byte order mark and replacement character.

By specifying these ranges, the CSS tells the browser exactly which characters it needs to display the page correctly using the chosen fonts, optimizing the loading process.


</details>